### ISSUE TITLE
[Feature Request] Preference for `Author Date` instead of `Commit Date`

### ISSUE THREAD

**Issue by CodeOptimist** [ForkIssues/TrackerWin#2213]
Particularly when interactively rebasing a lot, it's nice to see the author dates (original creation date).
(Would apply everywhere: commit log, blame, file history, etc.)

<sub>TortoiseHg->Sourcetree->Fork. Love the app. ❤️</sub>

**Comment by DanPristupov** [comment]
IR and commit log do show author date.

However, blame and file history show committer date because it's important to see the actual time when commit was applied. I think it makes sense because otherwise blame and history views could show older commits above the new ones making whole navigation confusing.

**Comment by CodeOptimist** [comment]
That's perfect. You know I think I wrote this down months ago in my to-do, and then figured out exactly what you just said and completely agreed. But today I was cleaning up my to-do and saw I wrote the dates were different between those views, but I didn't see it in my current repo so I thought my note was for SourceTree and that I should submit a request for Fork. 😂 But no, you nailed it and I forgot to delete my todo. 😂 Thanks!
